rust-toolchain.toml

```
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "1.56"
+components = ["rustfmt"]
```

We should probably use the minimal profile, for consistency with the previous configuration. It's also recommended for CI here: https://rust-lang.github.io/rustup/concepts/profiles.html

```suggestion
components = ["rustfmt"]
profile = "minimal"
```